src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -147,18 +150,12 @@ public void setPerson(Person target, Person editedPerson) {
     public void associateOrdersWithPersons() {
         for (CustomerOrder customerOrder : customerOrderList.getOrders()) {
             addressBook.findPersonByPhone(customerOrder.getPerson().getPhone())
-                    .ifPresent(person -> {
-                        customerOrder.setOriginalPerson(person);
-                        person.addOrder(customerOrder);
-                    });
+                    .ifPresent(person -> person.addOrder(customerOrder));
         }
 
         for (SupplyOrder supplyOrder : supplyOrderList.getOrders()) {
             addressBook.findPersonByPhone(supplyOrder.getPerson().getPhone())
-                    .ifPresent(person -> {
-                        supplyOrder.setOriginalPerson(person);
-                        person.addOrder(supplyOrder);
-                    });
+                    .ifPresent(person -> person.addOrder(supplyOrder));
```

Comment:
Think setoriginalperson should not be removed as it sets a pointer in the order object to the person object in addressbook during app initialization. Correct me if I am wrong